e2e-tests/cypress/integration/appeal-submission-tasklist.feature

```diff
@@ -0,0 +1,22 @@
+@wip
+Feature: Your Details status
+
+    Scenario Outline: The status of the Your Details section is accurate
+        # correct this, and correct your-details feature; 'valid' should not feature here; we want to be able to re-use for invalid cases too
```

Comment:
I am not sure we should have any comments in the feature file.

Reply:
we 100% shouldn't
however -  we're proposing that we'd like to re-use some existing sentences but the existing sentences aren't quite the right shape... if we're happy with the proposed syntax I'll pick this up and go re-work the existing sentences+glue so we get the re-use.. i just can't be bothered to do that until we know the new structure is 👍

Reply:
Np. thanks